src/tasks/handlers/taskHandler.ts

```diff
@@ -1,54 +1,73 @@
 import { BadRequestError } from '@map-colonies/error-types';
 import { Logger } from '@map-colonies/js-logger';
 import { IJobResponse, ITaskResponse, JobManagerClient, OperationStatus } from '@map-colonies/mc-priority-queue';
-import { IConfig, IJobDefinitionsConfig, JobAndTask, TaskTypeArray } from '../../common/interfaces';
+import { IConfig, IJobDefinitionsConfig, JobAndTask, TaskTypes, TaskTypeItem } from '../../common/interfaces';
 import { createTaskParametersMapper } from '../../common/mappers';
 
 /**
  * Utility class for workflow-specific task operations
  * This class is designed to be composed with job handlers to provide task-level functionality
  */
-export class WorkflowTaskOperations {
+export class TaskWorker {
```

Comment:
personally think task worker is terrible name, its taskHandler , you even named the file name `taskHandler.ts`

Reply:
furthermore, u named `jobHandler` so keep on same convention